samples/database-update-with-proto-descriptor.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2023 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+
+function main(
+  instanceId = 'my-instance',
+  databaseId = 'my-database',
+  projectId = 'my-project-id'
+) {
+  // [START spanner_update_database_with_proto_descriptors]
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance-id';
+  // const databaseId = 'my-database-id';
+
+  // Imports the Google Cloud client library
+  const {Spanner} = require('@google-cloud/spanner');
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+  // Gets a reference to a Cloud Spanner instance and a database. The database does not need to exist.
+  const instance = spanner.instance(instanceId);
+  const database = instance.database(databaseId);
+
+  async function updateDatabaseWithProtoDescriptor() {
+    // Reading proto descriptor file
+    const protoDescriptor = fs
+      .readFileSync('./resource/descriptors.pb')
+      .toString('base64');
+
+    // Updated a new database with an extra statements which will create
+    // proto bundle and tables with proto column
+    console.log(`Updating database ${database.formattedName_}.`);
+    const createProtoBundleStatement = `CREATE PROTO BUNDLE (
+            spanner.examples.music.SingerInfo,
+            spanner.examples.music.Genre,
+            )`;
+    const createSingersTableStatementStatement = `CREATE TABLE Singers (
+            SingerId   INT64 NOT NULL,
+            FirstName  STRING(1024),
+            LastName   STRING(1024),
+            SingerInfo spanner.examples.music.SingerInfo,
+            SingerGenre spanner.examples.music.Genre,
+            SingerInfoArray ARRAY<spanner.examples.music.SingerInfo>,
+            SingerGenreArray ARRAY<spanner.examples.music.Genre>,
+            ) PRIMARY KEY (SingerId)`;
+
+    // Creating empty database
+    await database.create();
```

Comment:
Move this statement out of the function (for example to line 42) so it does not seem to be part of the sample.